### PR TITLE
Docs/web twig icons story

### PR DESCRIPTION
**Pull request by literat**

### CODE REVIEW

File: packages/web-twig/src/Resources/icons/Icons.stories.twig [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+{% block content %}
+
+    <section class="docs-Section">
+
+        <div class="Grid Grid--cols-2 Grid--tablet--cols-4 Grid--desktop--cols-6 text-center my-1000">
+
+        {% for icon in icons %}
+
+            <div class="mb-600">
+            <Icon name={{ icon }} />
```
Comment (crishpeen) [inline packages/web-twig/src/Resources/icons/Icons.stories.twig:12]: indentation

File: packages/web-twig/src/Resources/icons/Icons.stories.twig [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+            <h3 class="mt-500">{{icon}}</h3>
```
Comment (crishpeen) [inline packages/web-twig/src/Resources/icons/Icons.stories.twig:14]:
```suggestion
                <h3 class="mt-500">{{icon}}</h3>
```
indentation

File: apps/web-twig-demo/src/Controller/IconsController.php [hunk 3]
```diff
@@ -0,0 +1,39 @@
+<?php
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class IconsController extends AbstractController
```
Comment (crishpeen) [inline apps/web-twig-demo/src/Controller/IconsController.php:8]: no tests for this controller?

Comment (literat) [reply apps/web-twig-demo/src/Controller/IconsController.php:8]: Demo app has no tests.

File: packages/web-twig/src/Resources/icons/Icons.stories.twig (hunk 2 above)
Comment (literat) [reply packages/web-twig/src/Resources/icons/Icons.stories.twig]: Fixed.

File: packages/web-twig/src/Resources/icons/Icons.stories.twig (hunk 1 above)
Comment (literat) [reply packages/web-twig/src/Resources/icons/Icons.stories.twig]: Fixed.

**Review by crishpeen — APPROVED** [review]

**Pull request merged**